### DISCUSSION TITLE
Bug in 'get_feed_result_document' decoding?

### DISCUSSION THREAD

**Discussion by dehidehidehi** [saleweaver/python-amazon-sp-api#103] [Q&A]
The string XML response had unexpected bytes at the end of the response, which tripped up my XML parser (lxml) see below.
b'\x0b'
![2021-03-23 12_14_04-Window](https://user-images.githubusercontent.com/40899632/112138036-51b25700-8bd1-11eb-866a-71f026d9a4b7.png)
```
<?xml version="1.0" encoding="UTF-8"?>
<AmazonEnvelope xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="amzn-envelope.xsd">
	<Header>
		<DocumentVersion>1.02</DocumentVersion>
		<MerchantIdentifier>[Removed]</MerchantIdentifier>
	</Header>
	<MessageType>ProcessingReport</MessageType>
	<Message>
		<MessageID>1</MessageID>
		<ProcessingReport>
			<DocumentTransactionID>201428018709</DocumentTransactionID>
			<StatusCode>Complete</StatusCode>
			<ProcessingSummary>
				<MessagesProcessed>455</MessagesProcessed>
				<MessagesSuccessful>455</MessagesSuccessful>
				<MessagesWithError>0</MessagesWithError>
				<MessagesWithWarning>0</MessagesWithWarning>
			</ProcessingSummary>
		</ProcessingReport>
	</Message>
</AmazonEnvelope>
```

**Comment by jakerator** [comment]
Confirming. I get the same at the end of XMLs. Have to cut them before parsing...
Looks like that comes from Amazon..

**Reply by saleweaver** [reply]
Can you please try this with the latest version? I think I've pushed a fix to this a week ago. 
This should have fixed it:

https://github.com/saleweaver/python-amazon-sp-api/commit/493209d2313678ca10ef748bd13706d719c367e1

**Reply by dehidehidehi** [reply]
Awesome, using 0.3.4 did fix the issue,
Thank you,
